packages/files/src/index.ts

```diff
@@ -70,7 +71,6 @@ export default (req: Request, res: Response, next: NextFunction) => {
         }
       }
     }
-    input = rr.combineDataKeys(data, keys);
```

Comment:
I think it was in the end so that we have also the content we read with `Object.assign(data, JSON.parse(content))` inside, do not move it to the beginning, if we put it there there was a reason. Read directly from data and keys